listenbrainz/db/dump.py

```diff
@@ -53,16 +54,16 @@
 # this dict contains the tables dumped in public dump as keys
 # and a tuple of columns that should be dumped as values
 PUBLIC_TABLES_DUMP = {
-    '"user"': (
+    'user': (
         'id',
         'created',
         'musicbrainz_id',
         'musicbrainz_row_id',
         # the following are dummy values for columns that we do not want to
         # dump in the public dump
-        'null',  # auth token
-        'to_timestamp(0)',  # last_login
-        'to_timestamp(0)',  # latest_import
+        SQL('null'),  # auth token
```

Comment:
should we add some docs here explaining the difference between strings and these specific objects?

Reply:
Added.